halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/services/v1/DaemonService.java

```diff
@@ -772,4 +772,7 @@
 
   @GET("/v1/spin/install/latest")
   DaemonTask<Halconfig, Object> installSpin();
+
+  @GET("/v1/spin/install/{version}")
```

Comment:
Does this cause ambiguity in the response mapping? Is it even worth keeping the /latest endpoint?

Reply:
This is a good point, `version=latest` is a real version. I can just version default to latest and simplify the controller, etc.